src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -705,7 +711,7 @@ class LicenceService(
       val additionalConditionUploadSummary = it.additionalConditionUploadSummary.map { upload ->
         upload.copy(id = -1)
       }
-      it.copy(id = -1, licence = newLicence, additionalConditionData = additionalConditionData, additionalConditionUploadSummary = additionalConditionUploadSummary)
+      it.copy(id = -1, licence = newLicence, additionalConditionData = additionalConditionData as MutableList<AdditionalConditionData>, additionalConditionUploadSummary = additionalConditionUploadSummary)
```

Comment:
can't just cast these if these are not the right type - not sure why these need to be mutable but if they do then you need to use `toMutableList()` instead